src/js/module/stats/statsUtil.js

```diff
@@ -0,0 +1,34 @@
+angular.module("gerritDashboard.stats").service("statsUtil",
+    [
+        function () {
+            "use strict";
+
+            var isInMap = function(map, key) {
```

Comment:
this one doesn't need to be an expression. You can declare a function. Not a big deal